src/Symfony/Component/HttpKernel/EventListener/AbstractTestSessionListener.php

```diff
@@ -47,7 +47,9 @@ public function onKernelRequest(RequestEvent $event)
 
         // bootstrap the session
         if (!$session = $this->getSession()) {
```

Comment:
shouldn't we deprecate `getSession()` in the test-listener?

Reply:
Good catch, I think we can even use the session inside the `request` by default, and keep `getSession` for BC